app/ios/Podfile.lock

```diff
@@ -1,4 +1,227 @@
 PODS:
+  - abseil/algorithm (0.20200225.0):
```

Comment:
Do you what are these pods for?  I've seen this before as part of my experimentation, but no idea how I got it

Reply:
Yeah, `Abseil` is a C++ library (funny name). https://cocoapods.org/pods/abseil 
 It's definitely required by FirebaseFirestore. https://cocoapods.org/pods/FirebaseFirestore (dependencies on the right side of the page)